.travis.yml

```diff
@@ -19,20 +19,13 @@ script:
 before_deploy:
   - yarn docs
 deploy:
-  - provider: npm
-    on:
-      branch: master
-    skip_cleanup: true
-    email: cicd@cognite.com
-    api_key:
-      secure: ZRfpzsn8Xgkrq706u+GWd+HcFztaFL+dyeCeU3aoperJ4/r8WHkJ4Kd/ZAOl8ZcOCKy71YQfCrxFBrR6qzm0Dxo96zFwM18xh/g0GmWyfesTNhEE05GKkVf1Jj4j66ZN4rZkt/TPkdrQbmEFABATY1kPljBxwstwHKAFBlGINd04X+wPRUl3kKWT1K3oEinRrzDLF78TcciIet7oa9atQb7JY8mhr/rtlzyVL7Yz9fwb5l+wxPJpqN4G8ZvknCVtO3bGybRQuE7xu+BJ8+CvKR67ui0zfuOIgyHZxUe31yxceADRnes+0kFtrOhnbpHJ+WtroA0HUlBEfZTamelTZUciyLic9z34An/yNyJvfcHA6vfz8dGrVwHMxtGNz9DPy16hajzoik5XJGKq5YFCNalb1go1t/M7HpHqG68MR117zpDvEX8dJQwGWdrctA9y7Q3TZa5Xw6wNlIbCX9/nc/WwnbO0tutXk8+yHQzxuylM4Gqm9SRxfuYXrsAqciMVGOMBGppq9pDzilWPQMmShacNa80iL86NU2nF7G6R7EAf/Z98XUPo5lNZcvaUNWwlPzXy30/vRHDHC+fjkqYkIkGcds9u/LGIPF+87zK1x+h0kW34C0/wDC52nExqBBTmoQ6tF62HvBwFaPmwIxNXhomv4FksizlLu+4Afe9R0gg=
-  - provider: npm
+  - provider: script
+    node_js: lts/*
     on:
       branch: v1
     skip_cleanup: true
-    email: cicd@cognite.com
-    api_key:
-      secure: ZRfpzsn8Xgkrq706u+GWd+HcFztaFL+dyeCeU3aoperJ4/r8WHkJ4Kd/ZAOl8ZcOCKy71YQfCrxFBrR6qzm0Dxo96zFwM18xh/g0GmWyfesTNhEE05GKkVf1Jj4j66ZN4rZkt/TPkdrQbmEFABATY1kPljBxwstwHKAFBlGINd04X+wPRUl3kKWT1K3oEinRrzDLF78TcciIet7oa9atQb7JY8mhr/rtlzyVL7Yz9fwb5l+wxPJpqN4G8ZvknCVtO3bGybRQuE7xu+BJ8+CvKR67ui0zfuOIgyHZxUe31yxceADRnes+0kFtrOhnbpHJ+WtroA0HUlBEfZTamelTZUciyLic9z34An/yNyJvfcHA6vfz8dGrVwHMxtGNz9DPy16hajzoik5XJGKq5YFCNalb1go1t/M7HpHqG68MR117zpDvEX8dJQwGWdrctA9y7Q3TZa5Xw6wNlIbCX9/nc/WwnbO0tutXk8+yHQzxuylM4Gqm9SRxfuYXrsAqciMVGOMBGppq9pDzilWPQMmShacNa80iL86NU2nF7G6R7EAf/Z98XUPo5lNZcvaUNWwlPzXy30/vRHDHC+fjkqYkIkGcds9u/LGIPF+87zK1x+h0kW34C0/wDC52nExqBBTmoQ6tF62HvBwFaPmwIxNXhomv4FksizlLu+4Afe9R0gg=
+    script:
+      - npx semantic-release
```

Comment:
Where does it get the npm credentials from 🤔 ? Since you remove the credentials from this PR

Reply:
Travis env variables